be/src/formats/parquet/null_column_reader.h

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include "formats/parquet/column_converter.h"
+#include "formats/parquet/column_reader.h"
+
+namespace starrocks::parquet {
+
+// Used to pad with nulls for schema evolved columns, as per Iceberg spec.
+class NullColumnReader final : public ColumnReader {
+public:
+    explicit NullColumnReader() {}
+    ~NullColumnReader() override = default;
+
+    Status read_range(const Range<uint64_t>& range, const Filter* filter, ColumnPtr& dst) override {
+        dst->append_nulls(range.span_size());
```

Comment:
How about change `NullColumnReader` -> `DefaultColumnReader`

And calling `dst->append_default(range.span_size())`

Because we may support none-nullable column in the future, append_default is better.

Reply:
Yes Iceberg 3 [will support default values](https://iceberg.apache.org/spec/#default-values). I see that `append_default` will append nulls if the column is a `NullableColumn`. I guess this should be safe since currently Iceberg will enforce that dst is a `NullableColumn`. ( dont' want to accidentally return e.g. an empty string instead of NULL for optional column).